app/addons/fauxton/navigation/components/NavLink.js

```diff
@@ -26,12 +26,17 @@ const NavLink = ({link, active, isMinimized}) => {
     {'faux-navbar--narrow': isMinimized}
   );
 
+  // const iconBadgeClass = link.badge ?  : '';
   return (
     <a className={linkClass} href={link.href} target={link.target ? '_blank' : null} data-bypass={link.target ? 'true' : null}>
       <div data-nav-name={link.title} className="faux-navbar__itemarea">
 
         {!!link.icon ?
-          <i className={classNames(link.icon, 'fonticon faux-navbar__icon')}></i> :
+          <i className={classNames(
```

Comment:
I know it was like this before, but what do you think about moving the this part out of the return and just rather have a variable we add here? I always find having an `if statement` in the JSX to be really confusing.